rasa/model.py

```diff
@@ -376,7 +376,7 @@ def _get_fingerprint_of_config_without_epochs(
     copied_config = copy.deepcopy(config)
 
     for key in ["pipeline", "policies"]:
-        if key in copied_config:
+        if key in copied_config and copied_config[key]:
```

Comment:
This is part of the other [PR](https://github.com/RasaHQ/rasa/pull/7482). It shows in the diff here because the PR isn't merged yet.
cc @joejuzl Can you address this in your original PR?